tree/ntuple/v7/inc/ROOT/RNTupleView.hxx

```diff
@@ -188,7 +191,13 @@ public:
 
    const RFieldBase &GetField() const { return *fField; }
    const RFieldBase::RValue &GetValue() const { return fValue; }
-   RNTupleGlobalRange GetFieldRange() const { return fFieldRange; }
+   RNTupleGlobalRange GetFieldRange() const
```

Comment:
Why do we want to throw in case of empty fields rather than returning an empty range?

Reply:
From my point of view, the idea is that there is a difference between an "empty field" that doesn't have members and where we cannot determine the field range, and an "empty field" that has an empty range (for example the item field of a `std::vector` that is always empty). For the invalid case, we should signal the user that they are doing something weird.

In any case, note that this is a backport which should go in as is. Changes to the logic should land in `master` first (and then be backported subsequently).

Reply:
Yes, I noticed afterwards that this was a backport and I missed the original PR. 
I guess the argument makes sense, although it's hard for me to guess the value-to-annoyance ratio from the user's perspective: if they have a loop over all fields where they create views, call GetFieldRange() and iterate over it, do they need to explicitly skip the invalid fields? Considering it's not an unrecoverable condition for us, would it be more frictionless to just do nothing, or would this lead to some incorrect state?
However, maybe let's discuss this offline - the backport PR is probably not the right place.

Reply:
I asked the same question, see #16768 for more context

Reply:
Thanks, sorry if I missed the conversation